unit/FArray/FArray.cpp

```diff
@@ -72,6 +72,7 @@ int main() {
     yakl::memset(test7d,0.f);
     yakl::memset(test8d,0.f);
 
+
```

Comment:
We need the same updates in `FArray` that we had in `CArray`.

Reply:
What do you mean by this?

Reply:
It might've been a bad github diff, but it looked like there were no OpenMP offload directives added to the FArray for loops in the serial routines.

Reply:
Wait, nevermind...my brain is fried.